app/src/test/java/com/github/h3lp3rs/h3lp/util/GDurationJSONParserTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.github.h3lp3rs.h3lp.util
+import com.github.h3lp3rs.h3lp.util.GDurationJSONParser
+import com.google.android.gms.maps.model.LatLng
+import junit.framework.Assert.assertEquals
+import org.json.JSONObject
+import org.junit.Test
+
+// Example of a time value returned by the directions api, it corresponds to 1 day, 3 hours and 2 mins
+// number of seconds
+const val NB_SECONDS = 97320
```

Comment:
These constants should be put inside the class I think in order to not pollute the other classes. I could be wrong but this is what transforms our files into .kt ones. This is actually worth the research. In Java for instance this is not possible (constants are put inside the class). Of course this does not concern only you but nearly all our files. Just mentioning :D